client/components/domains/register-domain-step/index.jsx

```diff
@@ -382,11 +387,11 @@ var RegisterDomainStep = React.createClass( {
 			},
 			suggestions = reject( this.state.searchResults, isSearchedDomain ),
 			availableDomain = find( this.state.searchResults, isSearchedDomain ),
-			onAddMapping = this.props.onAddMapping ?
-				domain => {
+			onAddMapping = this.props.onAddMapping
```

Comment:
Feels a bit too clever to the point where my IDE is complaining. I can wrap this in a simple if.